packages/prisma/extensions/booking-idempotency-key.ts

```diff
@@ -1,7 +1,8 @@
-import { Prisma } from "@prisma/client";
 import { v5 as uuidv5 } from "uuid";
 
-import { BookingStatus } from "@calcom/prisma/enums";
+import { Prisma } from "@calcom/prisma/client";
```

Comment:
Should we use relative paths since we're inside the prisma package here? @emrysal